### PR TITLE
[Nfs Ganesha] Add multi client scenarios

### PR DESCRIPTION
**Pull request by pranavprakash20**
Tests covered:
[1] Verify node reboot when cluister mounted on linux and windows clients [2] Verify kill of ganesha process while linux_untar in progress

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by pranavprakash20** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-jybong/NFS-Ganesha_multi_node_mount_with_v3_and_v4.1_and_perform_node_reboot_0.log

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-20NDS8/NFS-Ganesha_kill_ganesha_process_and_perform_failover_in_windows_client_0.log

**Review by MohitBis — DISMISSED** [review]
LGTM

File: suites/reef/nfs/tier1-nfs-ganesha-windows-clients_ha.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        install_packages:
+          - ceph-common
+          - ceph-fuse
+        copy_admin_keyring: true
+      desc: Configure the RGW,RBD client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
```
Comment (msainiRedhat) [inline suites/reef/nfs/tier1-nfs-ganesha-windows-clients_ha.yaml:100]: This is node reboot scenario, can we update the dec and name accordingly? Ganesha kill process scenario is not working as expected and we have BZ for same.

**Review by msainiRedhat — APPROVED** [review]
LGtM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**